catalog.json

```diff
@@ -2041,9 +2041,9 @@
             "home": "https://home.tdarr.io/",
             "location": "/__w/charts/charts/community/tdarr",
             "latest_version": "1.0.12",
-            "latest_app_version": "2.00.20.1",
```

Comment:
Please revert the changes in `catalog.json`

File is updated automatically after merge.
Thanks

Reply:
Done, thank you for the quick review!